busybox/19-enable-testcase-logging.patch

```diff
@@ -0,0 +1,23 @@
+--- busybox-1.27.2/testsuite/runtest	2023-10-27 17:17:52.404370639 +0200
++++ _phoenix/busybox-1.27.2/testsuite/runtest	2023-10-27 17:17:16.635573994 +0200
+@@ -40,7 +40,7 @@
+ 	mkdir -p ".tmpdir.$applet"
+ 	cd ".tmpdir.$applet" || return 1
+ 
+-#	echo "Running testcase $testcase"
++	echo "Running testcase $testcase"
+ 	d="$tsdir" \
+ 		sh -x -e "$testcase" >"$testname.stdout.txt" 2>&1 || status=$?
+ 	if [ $status -ne 0 ]; then
+
+--- busybox-1.27.2/testsuite/testing.sh	2023-10-27 17:17:52.404370639 +0200
++++ _phoenix/busybox-1.27.2/testsuite/testing.sh	2023-10-27 17:17:16.635573994 +0200
+@@ -87,7 +87,7 @@
+ 
+   $ECHO -ne "$3" > expected
+   $ECHO -ne "$4" > input
+-  [ -z "$VERBOSE" ] || echo ======================
++  echo "Running testcase $NAME"
+   [ -z "$VERBOSE" ] || echo "echo -ne '$4' >input"
+   [ -z "$VERBOSE" ] || echo "echo -ne '$5' | $2"
+   $ECHO -ne "$5" | eval "$2" > actual
```

Comment:
It seems that there are also other places where test cases start:
![Screenshot from 2023-11-08 12-35-25](https://github.com/phoenix-rtos/phoenix-rtos-ports/assets/77304431/4746609d-373f-44c8-9239-14f729b600ff)

There you also don't know when you should start parsing output.

Please correct it, run all the tests and make sure that all the output is formatted correctly.

Reply:
![image](https://github.com/phoenix-rtos/phoenix-rtos-ports/assets/116297248/26a2d286-64ae-417f-89e0-26c5326aa376)

Reply:
Thanks, one more nitpick: maybe it would be good to do  ---Testcase 'testcase name' starting---

Reply:
Did you mean that? 
![image](https://github.com/phoenix-rtos/phoenix-rtos-ports/assets/116297248/96ef2295-9d61-4b13-b5e0-cb62a73ce4e7)

Reply:
Isn't it the same?

Reply:
Is that what it was about?: 
![image](https://github.com/phoenix-rtos/phoenix-rtos-ports/assets/116297248/0725a085-da27-47c3-b1f6-4fca68b24890)

I didn't understand, previously I changed only that the testcase was printed and not the entire path in one case of printning logs.

Reply:
Yeah, exactly